### ISSUE TITLE
thank you for your release

### ISSUE THREAD

**Issue by fjptscottliu** [MengyuanChen21/CVPR2022-FTCL#3]
1.I check one of the issues,which random seed you choose?
2.Besides, ACMnet origin repo asked 1000 epochs training, your pre-train model set only 500,is that enough?
3.Can i change the random seed on exist best model?
looking forward to your reply
my current best mAP 38

**Comment by MengyuanChen21** [comment]
1. Zero. However, the same random seed will also lead to different results on different machines.
2. Yes, according to my experience, it is enough in most cases. The performance improvement after epoch 500 is marginal.
3. Sorry, I do not fully understand what you mean. After the training process ends, changing the random number seed will not affect the test results. However, you can try different random seeds to obtain better ACM-Net model. According to our experiments, the training of ACM-Net is not very stable with different random seeds. Therefore, before training the FTCL network, it should be guaranteed that ACM-Net can achieve the result (42.6% mAP@Avg) reported in its paper under the current random seed.

**Comment by fjptscottliu** [comment]
thank you! I almost have the same environment as you mentioned, but it seems  the pretrained  result is unpredictable,
if the pretrained ACMnet cannot reach 42.6,is it possible to reproduce FTCL result?

**Comment by MengyuanChen21** [comment]
I think that might be difficult to achieve. The results of FTCL are obtained when the pre-trained ACM-Net reaches 42.6. FTCL can generally improve the performance of existing methods, but its final result is still relevant to the backbone's performance. The discussions in the repository of [ACM-Net](https://github.com/ispc-lab/ACM-Net) might be helpful for you to reproduce their results.

**Comment by fjptscottliu** [comment]
thanks
